### ISSUE TITLE
[BUG]  left multiply of series by float isn't implemented.

### ISSUE THREAD

**Issue by jfpuget** [rapidsai/cudf#5590]
**Describe the bug**
a * s where a is float and s is a cuDF series isn't implemented.  s * a works fine.

**Steps/Code to reproduce bug**
Multiply a float by a cuDF series.

**Expected behavior**
No error.

**Environment overview (please complete the following information)**
 rapids 0.14

**Environment details**
rapids 0.14

**Additional context**
Very skinny description as the issue is very easy to reproduce.

**Comment by galipremsagar** [comment]
@jfpuget Looks like on `0.15` seems to work fine? or is the sample I'm trying different from what you have mentioned?

```python

>>> float_series = cudf.Series([1.2, 2.3])
>>> int_series = cudf.Series([1, 2])
>>> 1.5 * float_series
0    1.80
1    3.45
dtype: float64
>>> float_series * 1.5
0    1.80
1    3.45
dtype: float64

>>> int_series * 1.2
0    1.2
1    2.4
dtype: float64
>>> 1.2 * int_series
0    1.2
1    2.4
dtype: float64
```

**Comment by jfpuget** [comment]
It doe snot work with rapids 0.14.  Great if it is already fixed for 0.15.  I'll check that this example breads with .14.

**Comment by harrism** [comment]
Shall we close this?

**Comment by galipremsagar** [comment]
Closing this issue as it is fixed as of `0.15`. Feel free to report/reopen if this resurfaces.
